mmd_tools/operators/misc.py

```diff
@@ -50,27 +50,35 @@ def execute(self, context):
         obj = context.active_object
         root = mmd_model.Model.findRoot(obj)
         if root is None:
-            self.report({ 'ERROR' }, 'Select a MMD model') 
-            return { 'CANCELLED' } 
+            # self.report({ 'ERROR' }, 'Select a MMD model') 
+            # return { 'CANCELLED' }
+            # find all meshes in the active layer
+            act_layer = bpy.context.scene.active_layer
+            meshes_list = [o for o in bpy.data.objects
+                           if o.layers[act_layer] and o.type == 'MESH' and o.mmd_type == 'NONE']
+            active_mesh = meshes_list[0]
```

Comment:
There is an issue that if i seperated my mesh of mmd model by material, the related mesh property of material morphs will be out of sync when i join meshes to a normal cube by accident.

And i think it's more intuitive to use blender's tool `Ctrl-J` which can join selected meshes to active one. :smile:

Finally, it might be better to update the description of this operator since the behavior is a little different. :)